File: 27- আর্মস্ট্রং সংখ্যা.c
#include<stdio.h>
int main()
{

    int t;
    int x,x1;
    scanf("%d",&t);
    while(t--)
    {
        scanf("%d",&x);
        int m = x;
        int n=0;
        while(x!=0)
        {
            x1 = x%10;
            n = n+(x1*x1*x1);
            x = x/10;

        }
        if(n == m)
            printf("%d is an armstrong number!\n",m);
        else
            printf("%d is not an armstrong number!\n",m);

    }

    return 0;

}

File: 27- আর্মস্ট্রং সংখ্যা.py
t = int(input())

while t!=0:
    s = input()
    y = int(s)
    x = 0
    for i in s:
        n = int(i)
        x = x + pow(n, 3)
    if x==y:
        print(y,"is an armstrong number!")
    else:
        print(y,"is not an armstrong number!")

    t-=1


